starter/src/test/java/io/javaoperatorsdk/operator/springboot/starter/AutoConfigurationTest.java

```diff
@@ -26,18 +27,25 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.atIndex;
-import static org.junit.jupiter.api.Assertions.*;
-
-@ExtendWith(SpringExtension.class)
-@SpringBootTest
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.doNothing;
+
+@SpringBootTest(properties = {
+    "javaoperatorsdk.client.masterUrl=http://master.url",
+    "javaoperatorsdk.client.username=user",
+    "javaoperatorsdk.client.password=password",
+    "javaoperatorsdk.client.oauthToken=token"
```

Comment:
moved here because it interfered with integration tests, where these were provided by kubeconfig and the overridden